ddp-workspace/projects/ddp-pancan/src/app/components/footer/footer.component.scss

```diff
@@ -61,7 +61,7 @@
     min-width: fit-content;
 }
 
-@media screen and (min-width: $desktop-breakpoint) {
+@media screen and (min-width: $desktop-breakpoint + 1) {
```

Comment:
what do you think if we replace mobile first(min-width) with desktop first (max-width) here?

Reply:
Not sure if I get the point.
we already have 
```  
@media screen and (max-width: $desktop-breakpoint) {  // line 79
   ...change flex direction inside of some footer elements
} ```

The layout issue appears for the exact value `1024px`.

Reply:
I mean this
https://github.com/broadinstitute/ddp-angular/pull/816/files
what do you think?

Reply:
Deal, merged.
Will close this PR as redundant.